FlowControlHomework/FlowControlHomework/FlowControl.cs

```diff
@@ -357,7 +357,7 @@ public void Loops()
             // Dont change anything in the block, only the for statement.
             // hint: you dont need increment by 1.
             // This one is tough, take your time.
-            for (int i = 16; i < max2; i++)
+            for (int i = 0; i < max2; i+=2)
```

Comment:
What you had before (`int i = 16`) would have worked and technically followed the restrictions from the comments. But this is what I was expecting, and it's consistent with the "hint" from the comments.